src/patternfly/components/Card/card.scss

```
@@ -108,57 +136,128 @@
   --pf-c-card__header--m-toggle-right--toggle--MarginRight: calc(var(--pf-global--spacer--form-element) * -1);
   --pf-c-card__header--m-toggle-right--toggle--MarginLeft: var(--pf-global--spacer--xs);
 
-  // Card input
+  // Card input (selectable raised)
   --pf-c-card__input--focus--BorderWidth: var(--pf-global--BorderWidth--md);
   --pf-c-card__input--focus--BorderColor: var(--pf-global--primary-color--100);
 
+  position: relative;
   display: flex;
   flex-direction: column;
   background-color: var(--pf-c-card--BackgroundColor);
   box-shadow: var(--pf-c-card--BoxShadow);
 
-  &.pf-m-hoverable {
-    &:hover {
-      box-shadow: var(--pf-c-card--m-hoverable--hover--BoxShadow);
-    }
+  //////
+  // SELECTABLE CARDS, CLICKABLE CARDS
+  &.pf-m-selectable,
+  &.pf-m-clickable {
+    isolation: isolate;
+
+    // box-shadow will come from the card__label::before instead of the card so remove this so there's isn't a double shadow
+    box-shadow: none;
```

nit - we aren't consistent with this, but comments about a specific property should be inline with the property and it allows you to maintain the spacing

```suggestion
    isolation: isolate;
    box-shadow: none; // box-shadow will come from the card__label::before instead of the card so remove this so there's isn't a double shadow
```